### PR TITLE
Server 3 ga preview

### PR DESCRIPTION
**Pull request by annabaker**
Server 3 Install sections (pre-GA)

### CODE REVIEW

File: jekyll/_cci2/server-3-install-hardening-your-cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+| Nomad clients
+| Communication with the Nomad clients
+
+| all traffic
+| other nodes
+| Allow intra-cluster traffic
+|===
+
+## Nomad Clients
+Nomad clients do not need to communicate with each other, you can block traffic between Nomad client instances completely.
```
Comment (thomasr8) [inline jekyll/_cci2/server-3-install-hardening-your-cluster.adoc:138]:
```suggestion
Nomad clients do not need to communicate with each other; you can block traffic between Nomad client instances completely.
```

File: jekyll/_cci2/server-3-install-prerequisites.adoc [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+| 500+
+| 3
+| 48 cores
+| 240 GB
+| 10 Gbps
+|===
+--
+
+*** Your cluster must have outbound access to pull Docker containers and verify your license. If you don’t want to give open outbound access, see our https://help.replicated.com/community/t/customer-firewalls/55[list of ports] that will need access.
```
Comment (thomasr8) [inline jekyll/_cci2/server-3-install-prerequisites.adoc:47]:
```suggestion
*** Your cluster must have outbound access to pull Docker containers and verify your license. If you do not want to provide open outbound access, see our https://help.replicated.com/community/t/customer-firewalls/55[list of ports] that will need access.
```

File: jekyll/_cci2/server-3-install-prerequisites.adoc [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+* A CircleCI License file. Contact https://support.circleci.com/hc/en-us/requests/new[CircleCI support] for a license.
+* The <<Required Tools>> tools installed
+** Note that the `kubectl` tool must also be configured to have access your cluster
```
Comment (thomasr8) [inline jekyll/_cci2/server-3-install-prerequisites.adoc:50]:
```suggestion
** Note that the `kubectl` tool must also be configured to have access your cluster.
```

**Review by thomasr8 — COMMENTED** [review]
Looks really good @annabaker ! I made some minor suggestions that you may wish to incorporate in the doc.

File: jekyll/_cci2/server-3-install-hardening-your-cluster.adoc [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+This section provides supplemental information on hardening your Kubernetes cluster.
+
+toc::[]
+
+## Network Topology
+A server installation basically runs three different type of compute instances: The Kubernetes nodes, Nomad clients and external VMs.
+It is highly recommended to deploy these into separate subnets with distinct CIDR blocks. This will make it easier for you
+to control traffic between the different components of the system and isolate them from each other.
+
+As always, the rule is to make as many of the resources private as possible, applies. If your users will access your
```
[the remark this answers is not in the document]
Comment (teesloane) [reply jekyll/_cci2/server-3-install-hardening-your-cluster.adoc:22]: I think it's just a line break.

**Comment by teesloane** [comment]
@thomasr8 covered a lot, and I don't have much to add right now. Nice work. This is a heap of content!

**Review by teesloane — APPROVED** [review]

**Pull request merged**